### ISSUE TITLE
Central logging schema

### ISSUE THREAD

**Issue by joeph** [OraOpenSource/Logger#232]
Great functionality, thank you.

Is it possible to install the logger in a central schama and that the working schema's log to this central schema?

Cheers,
Joep

**Comment by martindsouza** [comment]
Yes. Just don't name the central schema `logger` as you'll have to use fully qualified names to reference `logger`.

https://github.com/OraOpenSource/Logger/blob/master/docs/Installation.md#restrict-access describes the installation practice and https://github.com/OraOpenSource/Logger/tree/master/source/scripts has the scripts needed to do this.

**Comment by joeph** [comment]
Thanks, how could I miss that. 
Would it be possible to have the schema name automatically be inserted into a column in the logging table. 

Background info: we have multiple schema's that own objects with identical names

**Comment by nicetheory** [comment]
I think the columns user_name, sid and client_info might help you to differentiate here.
